templates/fine-tune-llm_v2/end-to-end-examples/fine-tune-preference/README.ipynb

```diff
@@ -0,0 +1,1071 @@
+      "investigation was carried out D. She was transferred to a different department\n",
+      "E. She was promoted to a higher rank  Q3) Where were the photos of the police\n",
+      "officer performing a sex act in uniform initially taken? A. At a public park B.\n",
+      "At the police station C. At a private residence D. At a hotel E. At a restaurant\n",
+      "Q4) Why did the police officer, Cynthia Marrero Pomales, upload the pictures on\n",
+      "social media? A. To share with the public B. To share with the media C. To share\n",
+      "with her colleagues D. To share with a small private circle of friends E. To\n",
+      "share with her family  Q5) What was the reaction of the superintendent of police\n",
+      "in Puerto Rico, Jose Luis Caldero Lopez, to the incident? A. He denied the\n",
+      "authenticity of the images B. He resigned from his position C. He launched a\n",
+      "separate investigation D. He confirmed the authenticity of the images and\n",
+      "announced the suspension of the police officer E. He ignored the incident\n",
+      "\n",
+      "ANSWERS:\n",
+      "['B' 'C' 'B' 'D' 'D']\n",
+      "\n",
+      "'================================================================================'\n"
+     ]
+    }
+   ],
+   "source": [
+    "for row in example_rows:\n",
+    "    print_wrapped(\"TEXT\", row[DataSchema.ARTICLE])\n",
+    "    print_wrapped(\"QUESTIONS\", row[DataSchema.MCQ_QUESTIONS])\n",
+    "    print_wrapped(\"ANSWERS\", str(row[DataSchema.GROUND_TRUTH_MCQ_ANSWERS]))\n",
+    "    pprint.pprint(\"=\" * 80, width=80)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Part (b): Generate Summaries + Scores\n",
+    "\n",
+    "Next, we will generate 10 summaries for each article in the training set and score them with our Q&A judging setup. \n",
+    "\n",
+    "The following command will run the [generate_summaries_and_scores.py](src/scripts/generate_summaries_and_scores.py) script, which takes in the folder with generated questions + articles and stores the results to a new folder of `.parquet` files. This script will use the model under training to produce 10 summaries per each example on all of the input data examples. Followed by each summarization, it will also perform summary accuracy measurement, asking the down-stream LLM to answer the questions generated earlier solely based on the summaries generated by the desired model. \n",
+    "\n",
+    "🔄 REPLACE the S3 URI in [`configs/summary_generation/8b_judge/mistral_finetuned_eval.yaml`](configs/summary_generation/8b_judge/mistral_finetuned_eval.yaml) with the path to the folder with generated questions from the previous job\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This job will take ~320 min for 8B on 14 A10Gs (~ $76) and for 70B on A100s (~ $125) given the default configurations.\n",
```

Comment:
formatting?

Reply:
Yeah that's an issue with Github's markdown rendering for .ipynb. Check out the .md files!